action.yml

```diff
@@ -1,18 +1,18 @@
-name: 'Labeler'
-description: 'Automatically label new pull requests based on the paths of files being changed'
-author: 'GitHub'
+name: "Labeler"
```

Comment:
Would you mind to add the same [Prettier config](https://github.com/simple-icons/simple-icons/blob/develop/.prettierrc.json) of all repos? That will use single quotes in formatting.

Reply:
np. I've updated all files with single quote.